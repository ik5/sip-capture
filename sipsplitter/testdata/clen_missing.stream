INVITE foo@bar SIP/2.0

