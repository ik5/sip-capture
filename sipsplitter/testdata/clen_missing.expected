{
  "Original": "INVITE foo@bar SIP/2.0\r\n\r\n",
  "Discarded": 1,
  "NoStartLine": 0,
  "StartLine": 1,
  "NoHeaders": 0,
  "Header": 1,
  "NoBody": 0,
  "Body": 0,
  "Complete": 0,
  "Discards": [
    "INVITE foo@bar SIP/2.0\r\n\r\n"
  ],
  "Starts": [
    "INVITE foo@bar SIP/2.0\r\n"
  ],
  "Headers": [
    ""
  ],
  "Bodies": [],
  "Messages": []
}